docs/source/programming_resources/shared/configuring_android/Configuring-Your-Android-Devices.rst

```diff
@@ -55,7 +62,7 @@ smartphone for use as an DRIVER STATION. The process is as follows,
 Renaming Your Smartphones
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The official rules of the *FIRST* Tech Challenge (see <RS01>) require that
+The official rules of the *FIRST* Tech Challenge (see R707) require that
```

Comment:
Rephrase to not point to a specific rule number. Rule numbers may shift year to year.  

Suggest something like:

> Check the Robot Construction rules in the [Competition Manual](https://ftc-resources.firstinspires.org/file/ftc/game/manual) for details.